data/actions/transport.yaml

```diff
@@ -2,16 +2,9 @@ arrêter l'avion:
   icônes: 🛬🧳
   formule: transport . avion
   description: |
-    L'avion a une empreinte énorme sur le climat. 
-
-    On entend souvent que ramenée au passager, elle est du même ordre que celle d'une voiture. C'est vrai, mais ce n'est qu'une partie de l'histoire. 
-
-    Premièrement, l'empreinte de la voiture elle-même est élevée. Comparer l'avion à la voiture, ne rend pas ce premier vertueux.
-
-    Deuxièmement, l'empreinte climat est plus large que le seul CO2. C'est pour cela que l'on parle de gaz à effet de serre. L'empreinte totale de l'avion est estimée à 2x celle du CO2.
-
-    Finalement, on ne peut comparer la voiture et l'avion simplement parce qu'un trajet en avion est en général bien plus long. Quand il s'agit de choisir la destination des vacances par exemple, un trajet en voiture a donc toutes les chances d'être beaucoup moins polluant. Cela dit, notons aussi que nos kilométrages en voiture sont tels que le match est relancé quand on regarde l'empreinte à l'année.
-
+    L'avion a un impact important sur le climat à la fois en émissions de CO2 et d'autres effets (dit forçage radiatif additionnel) comme on peut le voir sur [le comparateur transport](https://ecolab.ademe.fr/apps/transport).
```

Comment:
Je trouve ta proposition de texte trop technique, en particulier introduire ici la notion de forçage radiatif additionnel. 
C'est ce que j'ai voulu expliquer plus simplement en disant que empreinte climat = CO2 + les autres gaz à effet de serre. 

Je pense aussi que la comparaison avec la voiture est intéressante à expliquer. Cela dit, on peut ajouter un attribut `aller plus loin` (il suffit de l'ajouter, ça n'aura aucun effet mais on pourra y mettre ce qu'on veut introduire dans l'interface par la suite dans une autre version) pour y mettre les informations plus techniques, précises, ou en relation plus indirecte.

Mieux vaut éviter de supprimer tout un paragraphe sans préciser pourquoi 😅 
Mieux vaut justifier

Reply:
En fait c'est le texte proche de l'autre comparateur [transport](https://ecolab.ademe.fr/transport). Donc on parle du phénomène tel qu'utilisé et mesuré par les études scientifiques.
Et sinon je ne suis pas fan de comparer à la voiture, c'est pas trop le sujet et ca nuit à la clareté du propos sur l'avion (sauf à y faire un paragraphe, mais pourquoi là et pas sur le reste, du coup ca sème le doute). Alimenter la FAQ pour ce point ?

Reply:
sans compter que parler voiture qd la personne parle avion, c'est étrange pour pas mal d'usage. Impossible d'aller en avion à NYC, ni à Casablanca...

Reply:
OK pour la voiture, je le stocke dans un autre attribut en attendant. 

Pour mon point sur la simplicité du texte, je pense que les fiches action ne peuvent pas être comparées aux textes sur le simulateur transport. Sur ce dernier, ce sont des sources techniques. Ici, c'est une explication simple de l'action destinée à tout citoyen.

Reply:
Bon, j'ai essayé de trouver un juste milieu : 889bccdd7d226f0e03757eeeacac3fd616ea8f85
Je fusionne comme ça et on peut évidemment itérer par la suite.